app/views/external/landing.html.haml

```diff
@@ -20,12 +20,20 @@
 #top-menu.top-menu= render "layouts/menu"
 
 .page-box
+  -if FALSE
```

Comment:
This only works because Ruby interprets the false as `nil`. To use `true` and `false` use lowercase `false` here. Also, add a space after the initial dash for consistency. `- if false`.

Reply:
Done